packages/snippets/bin/snippets.dart

```diff
@@ -224,10 +224,10 @@ void main(List<String> argList) {
   } else {
     final List<String> idParts = <String>[];
     if (packageName.isNotEmpty && packageName != 'flutter') {
-      idParts.add(packageName);
+      idParts.add(packageName.replaceAll(RegExp(r'\W'), '_').toLowerCase());
```

Comment:
> No need to add tests, since the functionality should be the same and existing tests cover it.

This does look like new functionality, though, that should probably be covered by a test?

Reply:
Oh yeah, good point.  OK, one test coming up!